travel/res_config.py

```diff
@@ -36,6 +38,26 @@ def get_basic_passenger_limit(ir_config_parameter_pool, cr, uid, context=None):
         return limit
 
 
+def get_alert_address(ir_config_parameter_pool, cr, uid, context=None):
+    context = context or {}
```

Comment:
Not exactly the same if the passed context is {}, you're overwriting the context.